beta-src/src/state/game/game-api-slice.ts

```diff
@@ -380,22 +424,26 @@ const gameApiSlice = createSlice({
     processUnitClick(state, clickData) {
       const {
         data: {
-          data: { contextVars },
-        },
-        order: {
-          inProgress,
-          method,
-          onTerritory,
-          orderID,
-          toTerritory,
-          type,
-          unitID,
+          data: { contextVars, currentOrders },
         },
+        order: { inProgress, method, onTerritory, orderID, type, unitID },
         ownUnits,
+        ordersMeta,
       } = current(state);
       if (contextVars?.context?.orderStatus) {
         const orderStates = getOrderStates(contextVars?.context?.orderStatus);
-        if (orderStates.Ready) {
+        const clickedUnit = clickData.payload.unitID;
```

Comment:
you can destructure `const { unitID }  = clickData.payload` or just inline clickData.payload.unitID on L437